pkgs/development/python-modules/poetry/default.nix

```diff
@@ -0,0 +1,68 @@
+{ lib, buildPythonPackage, fetchPypi, callPackage
+, isPy27, isPy34
+, cleo
+, requests
+, cachy
+, requests-toolbelt
+, pyrsistent
+, pyparsing
+, cachecontrol
+, pkginfo
+, html5lib
+, shellingham
+, tomlkit
+, typing
+, pathlib2
+, virtualenv
+, functools32
+}:
+
+let
+  cleo6 = cleo.overrideAttrs (oldAttrs: rec {
+  version = "0.6.8";
```

Comment:
please indent properly here. Unfortunately I don't know of a proper autofixing tool for Nix CS, so we have to fix this manually :(